_posts/2024-02-28-the-return-of-the-golden-man.md

```diff
@@ -0,0 +1,11 @@
+---
+title: The return of the golden <s>child</s> man
```

Comment:
**issue:** I didn't get that reference and it looks a bit odd. Might want to reconsider that.
IMO the PR title would make good one for the blog post: *The return of Julien*

Reply:
totally right. The return of *the* Julien?